backend/operations_scanner.go

```diff
@@ -567,3 +575,49 @@ func convertClusterStatus(clusterStatus *arohcpv1alpha1.ClusterStatus, current a
 
 	return opStatus, opError, err
 }
+
+func (s *OperationsScanner) convertInflightChecks(ctx context.Context, internalId ocm.InternalID) (*arm.CloudErrorBody, error) {
+	inflightChecks, err := s.clusterService.GetClusterInflightChecks(ctx, internalId)
+	if err != nil {
+		return &arm.CloudErrorBody{}, err
+	}
+
+	var cloudErrors []arm.CloudErrorBody
+	for _, inflightCheck := range inflightChecks.Items() {
+		if inflightCheck.State() == "failed" {
```

Comment:
Probably should use the `InflightCheckStateFailed` constant here.

Reply:
I couldn't find an existing constant for that, so I created one here https://github.com/Azure/ARO-HCP/pull/1646/files#diff-8e0a776b7f44845193da0fd0a483d869eceaa15415306a81b85b647292d7a306R62, please let me know if that's the right place.